internal/pkg/cli/interfaces.go

```diff
@@ -531,6 +531,20 @@ type serviceLinkedRoleCreator interface {
 	CreateECSServiceLinkedRole() error
 }
 
+type roleTagsLister interface {
+	ListRoleTags(string) (map[string]string, error)
+}
+
+type roleManager interface {
+	roleTagsLister
+	roleDeleter
+	serviceLinkedRoleCreator
+}
+
+type stackExister interface {
```

Comment:
nit: Is there a way we can reframe this? The name is a little confusing, and while it's clear that it's not a `stackCreator` it's not immediately clear what the interface does from the `exister` verb.
```suggestion
type stackExistChecker interface 
```

Reply:
Done!